domains/transfer/TransferAgreementAutomation/Worker/Metrics/TransferAgreementAutomationMetrics.cs

```diff
@@ -1,7 +1,7 @@
 using System.Collections.Generic;
 using System.Diagnostics.Metrics;
 
-namespace API.Transfer.TransferAgreementsAutomation.Metrics;
+namespace TransferAgreementAutomation.Worker.Metrics;
 
 public class TransferAgreementAutomationMetrics : ITransferAgreementAutomationMetrics
```

Comment:
Did you check that the metrics are published to the existing metrics?

Reply:
Good point.
Just created a TA between peter producent and erik energinet. I. will follow up on next run.